src/containers/EmailVerification.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { ClipLoader } from 'react-spinners';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import { SectionHeader, SectionText } from '../styled/headers';
+import { PageContainer, SectionHeaderContainer } from '../styled/containers';
+import { StyledButton } from '../components/common';
+import { emailVerification, resendEmailVerification } from '../actions';
+
+const EmailVerification = ({ location: { search } }) => {
+  const emailVerificationState = useSelector(state => state.emailVerification);
+  const token = search.replace('?token=', '');
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(emailVerification(token));
+  }, []);
+
+  const resendVerification = () => {
+    dispatch(resendEmailVerification(token));
```

Comment:
you're dispatching the token, which seems correct, but is the token supposed to be attached to a link in the backend somehow? this is the email i get from the verification system and I'm not sure what I'm supposed to do with the token without a link. Also needs a `\n` after the token itself.

![image](https://user-images.githubusercontent.com/7741746/68449740-3cd05180-019d-11ea-968f-ae4800cd1240.png)

Reply:
Originally the frontend didn't have a component to fire off the post request, now that  I've created the email verification component now I can make the backend send a link instead of a token.

Reply:
Gotcha - can you implement sending a link then? I can't really do anything with a token + I'm not sure where the token goes (eg is it slohacks.com/?token=x, or slohacks.com/#/confirm?token=x, etc)